testsuite/oidc/rhsso/objects.py

```diff
@@ -49,15 +49,6 @@ def create_realm_role(self, role_name: str):
         role_id = self.admin.get_realm_role(role_name)["id"]
         return {"name": role_name, "id": role_id}
 
-    def assign_realm_role(self, role, user_id: str):
```

Comment:
there is one more usage of this method in the `rhsso.test_rhsso_roles.py::user_with_role` fixture